1_BayesIntuition.ipynb

```diff
@@ -0,0 +1,893 @@
+{
```

Comment:
<ul><li>defending?</li><li>from 0 to 1</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/canyon289/pyladies-bayes/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>